mesonbuild/modules/cmake.py

```diff
@@ -33,8 +34,18 @@
 
     InvalidArguments,
     InterpreterException,
+
+    typed_kwargs,
+    KwargInfo,
 )
 
+class WriteBasicPackageVersionFile(T.TypedDict):
```

Comment:
This should only be defined when `if T.TYPE_CHECKING:`, see CI failures.

Reply:
In other words, this works on python 3.8 but errors out on python 3.7, and once moved into a mypy-only code block we can simply not care that it doesn't support using mypy which itself runs on python 3.7

Reply:
Done, thanks :)
I wasn't really sure how to fix that